tests/sqllogictests/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "paradedb-sqllogictest"
```

Comment:
Why do we need a separate crate? Can we make use of the `tests` crate instead?

Reply:
Separating the `sqllogictest `  from the test crate is better for the organization because it keeps different types of tests isolated, making the structure cleaner and easier to manage. I prefer to isolate it from the test crate.

Reply:
A subfolder make sense, but why does it need to be its own crate?

Reply:
If we isolate it from the test crate, we can easily run `cargo test --package paradedb-sqllogictest --features pg17`, which separates the test result from `rstest`. I prefer this organization, but if you insist, I don't mind keeping `sqllogictest` under the test crate.

Reply:
If you prefer this organization it's fine. I just wanted to understand better :)